docs/team/anqi20.md

```diff
@@ -4,22 +4,33 @@
 
 E-Duke-8 (pronounced "Educate") helps CS2113/T students learn and understand software engineering and Object-Oriented Programming (OOP) principles through a gamified platform and enhances their learning experience. 
 
-### Summary of Contributions
+## Summary of Contributions
 
 - **New Feature:** Implemented the timer function for the quiz
   - What it does: The user would be able to choose the timer set for each question. If the time is up, and the user have yet to complete the question, then it will be marked as an incorrect answer. 
-  - Justification: E-Duke-8 is able to simulate a timed examination environment for the students. 
-- **New Feature:** Added the need for users to press "Enter" before they can proceed to the next question or end the quiz
-  - What it does: 
-  - Justification: 
+  - Justification: E-Duke-8 is able to simulate a timed examination environment for the students. Moreover, since they get to choose the timing , E-Duke-8 is able to cater to all students at different learning stages. 
+  - Highlights: This timer feature is challenging as we had to cater to users using different operating systems. Kingston has helped in implementing this feature for the Windows Operating System. 
+- **New Feature:** Added delay between each question during the quiz
```

Comment:
Looks good. I believe you also worked on Ui and some parts of the Model, do you want to mention those as well?

Reply:
Yep, I have updated the PPP as well. Thank you for the comments.